avocado/core/settings.py

```diff
@@ -182,21 +182,20 @@ def __init__(self, config_path=None):
                                           config_path_local,
                                           config_path_intree,
                                           config_path_pkg])
+            # First try in-tree config
             if config_intree:
-                # In this case, respect only the intree config
                 self.process_config_path(config_path_intree)
                 if config_intree_extra:
                     for extra_file in glob.glob(os.path.join(_config_path_intree_extra, '*.conf')):
                         self.process_config_path(extra_file)
                 self.intree = True
-            else:
-                # In this case, load first the global config, then the
-                # local config overrides the global one
-                if config_system:
-                    self.process_config_path(config_path_system)
-                    if config_system_extra:
-                        for extra_file in glob.glob(os.path.join(_config_dir_system_extra, '*.conf')):
-                            self.process_config_path(extra_file)
+            # Override with system config
+            if config_system:
+                self.process_config_path(config_path_system)
+                if config_system_extra:
+                    for extra_file in glob.glob(os.path.join(_config_dir_system_extra, '*.conf')):
+                        self.process_config_path(extra_file)
+            # And the local config
```

Comment:
Ok, so the `config_intree` will be the first one to be used, then overridden by the `config_system` and then overridden by the `config_path_local`. It makes more sense then to use `config_system` only if `config_intree` is not present. Testing this.